src-web/components/kappnav/modals/ActionModal.js

```diff
@@ -154,6 +154,14 @@ export const transform = (field, optional, handleChange, event) => {
   handleChange(field, optional, event.target.value)
 }
 
+export const transformMultiSelect = (field, optional, handleChange, event) => {
+  var selectedItemsValue = ''
+  event.selectedItems.map((selectedItem) => {
+    selectedItemsValue += selectedItem.text + ','
```

Comment:
Will this result in a tailing ,?  a,b,c,

Reply:
No, Katie, I have used a Regex to eliminate the trailing comma in the next line.
The result is like:
{multiselect-string: "First,Second,Third"}

Reply:
thanks!